modules/localstack/examples_test.go

```diff
@@ -53,9 +53,16 @@ func ExampleRun_withNetwork() {
 
 	newNetwork, err := network.New(ctx)
 	if err != nil {
-		log.Fatalf("failed to create network: %s", err)
+		log.Printf("failed to create network: %s", err)
+		return
 	}
 
+	defer func() {
+		if err := newNetwork.Remove(context.Background()); err != nil {
```

Comment:
Do you think we need TerminateNetwork function for the examples?

Reply:
I don't think so as we only need it for containers as it can be `nil` and can race so need the extra error handling. I've not seen that for networks, if we do lets do it seperately.